features/org.hibernate.search.eclipse.feature/feature.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="org.hibernate.search.eclipse.feature"
+      label="%featureName"
+      version="5.2.0.qualifier"
+      provider-name="%providerName"
+      plugin="org.jboss.tools.hibernate.search"
+      license-feature="org.jboss.tools.foundation.license.feature"
+      license-feature-version="0.0.0">
+
+   <description url="http://www.example.com/description">
+      %description
+   </description>
+
+   <copyright>
+      %copyright
+   </copyright>
+
+   <license url="http://www.eclipse.org/legal/epl-v10.html">
+      This program and the accompanying materials are made available under the terms of the Eclipse Public License v1.0 which accompanies this distribution.
+Also this plugin is an addition to Jboss Tools Hibernate plugin which is licensed under certain open source licenses (or aggregations of such licenses), which in a particular case may include the Eclipse Public License, the GNU Lesser General Public License, and/or certain other open source licenses
+   </license>
+
+   <requires>
+      <import plugin="org.eclipse.core.runtime" version="3.12.0" match="greaterOrEqual"/>
+      <import plugin="org.eclipse.ui.workbench"/>
+      <import plugin="org.eclipse.jdt.core"/>
+      <import plugin="org.eclipse.datatools.connectivity"/>
+      <import plugin="org.hibernate.eclipse"/>
+      <import plugin="org.eclipse.osgi"/>
+      <import plugin="org.eclipse.jface"/>
+      <import plugin="org.eclipse.datatools.connectivity.ui"/>
+      <import plugin="org.eclipse.debug.core"/>
+      <import plugin="org.hibernate.eclipse.console" version="5.1.2" match="greaterOrEqual"/>
+      <import plugin="org.jboss.tools.hibernate.runtime.spi" version="5.1.2" match="greaterOrEqual"/>
+      <import plugin="org.jboss.tools.hibernate.runtime.v_4_0" version="5.1.2.qualifier" match="greaterOrEqual"/>
```

Comment:
yes. My hibernate search extension works with all the same versions as hibernate plugin works with, except 3.6 - the earliest one, that's why i didn't include it here